bolt.gemspec

```diff
@@ -44,6 +44,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "puppet", [">= 6.0.1", "< 7"]
   spec.add_dependency "puppet-resource_api"
   spec.add_dependency "r10k", "~> 2.6"
+  spec.add_dependency "ruby_smb", "~> 1.0"
```

Comment:
BSD 3 clause license: https://github.com/rapid7/ruby_smb/blob/master/LICENSE.txt. That's probably acceptable.

Reply:
Filed as PE-25855